Components/Card/card.css

```diff
@@ -63,11 +97,22 @@ section .example-div {
     box-shadow: 3px 3px 20px 5px var(--black);
```

Comment:
suggestion: make utility class for shadow so that you can reuse it

Reply:
Cool. Will add this 👍🏻